Content.Shared/Climbing/BonkableComponent.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared.Climbing
+{
+    [RegisterComponent, NetworkedComponent]
+    public sealed class BonkableComponent : Component
```

Comment:
file scoped namespace to reduce indentation and once you move the system to shared add this attribute `[Access(typeof(BonkableSystem)] `